rvv-intrinsic-generator/rvv_intrinsic_gen/generator.py

```diff
@@ -397,6 +407,59 @@ def output_call_arg(arg_name, type_name):
     self.fd.write(call_args)
     self.fd.write(");\n")
     self.fd.write("}\n\n")
+    self.fd.flush() # To make sure the data flushed when post_gen.
+
+
+  def post_gen(self):
+    if self.gnu:
+      for test_file in set(self.test_files):
+        fd = open(os.path.join(self.folder, test_file), "r", encoding="utf-8")
+        api_count = fd.read().count("__riscv_")
+        fd.close()
+
+        opcode = test_file.removesuffix(".c")
+
+        # TODO: move to switch case if python version >= 3.10
+        if "_" in opcode:
+          pattern = opcode.replace("_", "\.")
```

Comment:
nit: maybe `pattern_string` / `pattern_str`, since you are not actually using the pattern string for regex search here in python.

Reply:
sure.